rotkehlchen/rotkehlchen.py

```diff
@@ -964,7 +974,12 @@ def query_asset_movements(
                     )
         else:
             # we may have DB entries due to csv import from supported locations
-            for external_location in EXTERNAL_EXCHANGES:
+            # Connected exchanges via API keys
+            exchanges = self.exchange_manager.connected_exchanges.keys()
+            # Exchanges added by importing from CSV file
```

Comment:
same comment about comments as above. Also this code seems to be duplicating. Perhaps abstract it in a function?